modules/improvedigitalBidAdapter.js

```diff
@@ -0,0 +1,386 @@
+const LIB_VERSION_GLOBAL = '3.0.5';
+
+var CONSTANTS = require('src/constants');
+var utils = require('src/utils');
+var bidfactory = require('src/bidfactory');
+var bidmanager = require('src/bidmanager');
+var adloader = require('src/adloader');
+var Adapter = require('src/adapter').default;
+var adaptermanager = require('src/adaptermanager');
+
+const IMPROVE_DIGITAL_BIDDER_CODE = 'improvedigital';
+
+const ImproveDigitalAdapter = function () {
+  let baseAdapter = new Adapter(IMPROVE_DIGITAL_BIDDER_CODE);
+  baseAdapter.idClient = new ImproveDigitalAdServerJSClient('hb');
+
+  let LIB_VERSION = LIB_VERSION_GLOBAL;
+
+  // Ad server needs to implement JSONP using this function as the callback
+  let CALLBACK_FUNCTION = '$$PREBID_GLOBAL$$' + '.improveDigitalResponse';
```

Comment:
FYI we won't allow JSONP callbacks with prebid 1.0.

Reply:
We support CORS and will update our adapter when required by prebid 1.0.